src/eckit/maths/CMakeLists.txt

```diff
@@ -1,25 +1,26 @@
-list(APPEND eckit_maths_private_libs "${LAPACK_LIBRARIES}" "${BLAS_LIBRARIES}")
-list(APPEND eckit_maths_sources
+list( APPEND eckit_maths_lib_srcs
     Eigen.h
-    Lapack.h
     Lapack.cc
+    Lapack.h
     Matrix.h
     MatrixEigen.h
-    MatrixLapack.h)
+    MatrixLapack.h
+)
 
-if(eckit_HAVE_CONVEX_HULL)
-    list(APPEND eckit_maths_sources ConvexHull.h ConvexHullN.h Qhull.cc Qhull.h)
-    list(APPEND eckit_maths_private_libs Qhull::Qhull)
+if( eckit_HAVE_ECKIT_GEO )
+    # for eckit::geo
+    list( APPEND eckit_maths_lib_srcs
+        Matrix3.h
```

Comment:
Is this Matrix type really _only_ applicable with eckit geo?

Reply:
No, it has more genral use, I was containing the changes limited to the library. I'll include this uncondittionally